src/guides/v2.3/ui_comp_guide/howto/new_component_declaration.md

```diff
@@ -46,88 +46,7 @@ If the custom component you create is a [basic UI component] (like Form or Listi
 1. Declare the component in a separate `.xml` file using the `<container>` or `<component>` as parent node.
 
 ## Declare a custom secondary component
```

Comment:
I would recommend removing this section completely, without mention that the secondary components are not supported. We don't define what the "secondary component" actually means. So the message about the unsupported components might be confusing.

Reply:
@rogyar  I had thought about that, but as this has been there for so long, some developers might know about it and  query where it had gone. So it was left there for this reason. What do you think?

Reply:
Hi @BarnyShergold. That makes sense, you are right. I think in both cases it might be confusing either for new devs or for devs who saw this part of the topic before. So, we may leave it as it is

Reply:
@rogyar - Can you approve the PR then please? 😸